pkg/resource/lateinit.go

```diff
@@ -230,6 +262,16 @@ func (li *GenericLateInitializer) handleStruct(parentName string, desiredObject
 			continue
 		}
 
+		for _, f := range li.conditionalFilters {
+			if f(cName) {
```

Comment:
If we follow the above suggestions, this would turn into something like:
```suggestion
			if f(cName, desiredPaved, actualPaved) {
```